platform/web/jsonschema.go

```diff
@@ -0,0 +1,92 @@
+package web
```

Comment:
Does this really belong in the "web" package you think? 

I guess you made this function because we could re-use it elsewhere, but the kind of nice thing about this abstraction is that it's not tied anymore to HTTP requests, so I am not sure it should be kept in the web package.